### PR TITLE
Bf branch

### PR DESCRIPTION
**Pull request by CreativeFitz**
# Description
Added classes for Employee, ProductType and Training Program.

## Steps to Test
Clone repo down to local branch.
Open code for BangazonAPI in Visual Studio.
Check models folder for Employee, Training, and ProductType classes.

# Checklist:
- [N/A] I have commented my code, particularly in hard-to-understand areas
- [N/A ] I have made corresponding changes to the documentation
- [ Yes] My changes generate no new errors
- [N/A ] I have added tests that prove my fix is effective or that my feature works
- [N/A ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by BelcherCharles** [comment]
🔥

File: BangazonAPI/BangazonAPI/Models/Employee.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    public class Employee
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        public int DepartmentId { get; set; }
+        public bool IsSupervisor { get; set; }
+
```
Comment (kimpreece) [inline BangazonAPI/BangazonAPI/Models/Employee.cs]: According to ticket #6 a representation of the computer that the employee is currently using should be included in the employee representation".

**Review by kimpreece — DISMISSED** [review]

File: BangazonAPI/BangazonAPI/Models/Employee.cs (hunk 1 above)
Comment (Downeastrussell) [reply BangazonAPI/BangazonAPI/Models/Employee.cs]: 🍝

Comment (CreativeFitz) [reply BangazonAPI/BangazonAPI/Models/Employee.cs]: -I added an instance of computer as a property to employee.
-Also added an empty class called computer.

**Review by jordan-castelloe — APPROVED** [review]
🌮

**Pull request merged**